src/util/store.ts

```diff
@@ -1,32 +1,29 @@
-import { Store, applyMiddleware, compose, createStore } from "redux";
-import { thunk } from "redux-thunk";
+import { configureStore, EnhancedStore } from '@reduxjs/toolkit';
 import windyStore from "@windy/store";
 import * as skewTAct from "../actions/skewt";
 import * as soundingAct from "../actions/sounding";
-
-
 import { rootReducer } from "../reducers/sounding";
 
+export type AppStore = EnhancedStore<{ [K in keyof typeof rootReducer]: ReturnType<typeof rootReducer[K]> }>;
 
-let store: Store;
+let store: AppStore;
 
-export function getStore(container: HTMLDivElement) {
+export function getStore(container: HTMLDivElement): AppStore {
   if (store) {
     return store;
   }
 
-  const middlewares = [thunk];
-  const composeEnhancers =
-    (process.env.NODE_ENV == "development" ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ : null) ||
-    compose;
-    // TODO: check deprecated
-  store = createStore(rootReducer, composeEnhancers(applyMiddleware(...middlewares)));
+  // Automatically adds thunk and Redux DevTools
+  store = configureStore({
+    reducer: rootReducer,
+    middleware: (getDefaultMiddleware) => getDefaultMiddleware(),
```

Comment:
this should not be needed because it's the default :)

see https://redux-toolkit.js.org/api/getDefaultMiddleware#intended-usage

May be add:

`devTools: process.env.NODE_ENV !== 'production',`

as in devTools: process.env.NODE_ENV !== 'production',

Reply:
Oh, yes. I didn't mean to leave that included. Sorry! Let me fix that

Reply:
As for the `devTools`, I thought that was handled by the toolkit. Will fix it.

Reply:
I squashed the commits, if that is what you were wanting. I hope this looks better.